tools/perfscale-load-generator/README.md

```diff
@@ -0,0 +1,82 @@
+  - name: kubevirt-density-10
+    // iterationCount defined how many times to execute the workload
+    iterationCount: 1
+    // iterationInterval defines how much time to wait between each workload iteration
+    iterationInterval: 0
+    // iterationCreationWait wait for all objects to be running before moving forward to the next iteration
+    iterationCreationWait: true
+    // iterationCleanup clean up old tests, e.g., namespaces, nodes, configurations, before moving forward to the next iteration
+    iterationCleanup: true
+    // iterationDeletionWait wait for objects to be deleted in each iteration
+    iterationDeletionWait: true
+    // create a namespace per workload iteratio
+    namespacedIterations: false   
+    // maximum wait period for all iterations
+    maxWaitTimeout: 30m
+    // qps is the max number of queries per second to control the job creation rate
+    qps: 20
+    // burst is the maximum burst for throttle to control the job creation rate
+    burst: 20
+    // waitWhenFinished delays the termination of the workloa
+    waitWhenFinished: 30s
+    // objects defines a list of object spec to be created
+    objects:
+        // templateFile is the relative path to a valid YAML definition of a kubevirt resource
+      - templateFile: templates/vmi-ephemeral.yaml
+        // replicas is the number of replicas to create of the given object
+        replicas: 1
+        // InputVars contains a map of arbitrary user-define input variables that can be introduced in the template by users
+        // All the variables in the inputVars depend on the dynamic user-define variables in the templateFile
+        inputVars:
+          // containerPrefix defines the repository prefix for all images
+          // if the value is empty (i.e., "") the default value defined in the cmd flag will be used
+          containerPrefix: registry:5000/kubevirt/
+          // containerImg defines the VMI image in the 
+          containerImg: cirros-container-disk-demo
+          // containerTag defines the image tag
+          // if the value is empty (i.e., "") the default value defined in the cmd flag will be used
+          containerTag: devel
+          // namespace defines the prefix of the namespace to create the object
+          namespace: kubevirt-density
+```
+
+## Workflow
+- The load generator ranges the list of workloads and sequentially executes each workload.
+- Each workload might run one or more times in different iterations
+- Each workload iteration might create a list of different objects
+- Each workload iteration might wait for all objects to be created
+- Each workload iteration might clean up the iteration, deleting all created namespaces and consequently deleting all created objects
+- Each workload iteration might sleep X times before each iteration to avoid being affected by a previous execution
+
+## Comments
+The Objects are not directly deleted, they are indirectly deleted when the namespaces are deleted.
```

Comment:
is this still the case about indirectly being deleted?

Reply:
right, it is old and I removed it!